proto/quicksilver/interchainquery/v1/query.proto

```diff
@@ -16,7 +16,8 @@ option go_package = "github.com/ingenuity-build/quicksilver/x/interchainquery/ty
 service QuerySrvr {
   // Params returns the total set of minting parameters.
   rpc Queries(QueryRequestsRequest) returns (QueryRequestsResponse) {
-    option (google.api.http).get = "/quicksilver/interchainquery/v1/queries/{chain_id}";
+    option (google.api.http).get =
```

Comment:
@aljo242 - not sure why but something somewhere is causing Mukund and I to format proto in one way, and you another way, so we are constantly changing these lines back and forth between the two. Would be good to understand why this is happening!

Reply:
@joe-bowman @muku314115 what is your `buf` version on your local machine?

Reply:
yeah i would move to latest which is `1.21.0`

I think that is causing the issue